### PR TITLE
Remove duplicated code from openchemistrypy

### PR DESCRIPTION
**Pull request by alesgenova**
Related to https://github.com/OpenChemistry/openchemistrypy/pull/24

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -12,7 +12,7 @@ jobs:
       - checkout
       - run:
           name: Install requirements
-          command: pip install -r requirements.txt
+          command: pip install -r requirements.txt --process-dependency-links
```
Comment (cjh1) [inline .circleci/config.yml]: I think this pip option is being deprecated, we need it?

File: girder/molecules/server/calculation.py [hunk 2]
```diff
@@ -296,15 +298,22 @@ def create_calc(self, params):
...
             calc_data = f.read().decode()
-            if code == 'nwchem':
-                cjson = avogadro.convert_str(calc_data, 'json', 'cjson')
-                cjson = json.loads(cjson)
-            elif code == 'psi4':
-                cjson = cclib.convert_str(calc_data)
-            else:
-                raise Exception('Unknown code %s' % code)
+
+        with tempfile.TemporaryFile('w+') as tf:
```
Comment (cjh1) [inline girder/molecules/server/calculation.py]: Couldn't we do this in memory ```BinaryIO```?

File: requirements.txt [hunk 3]
```diff
@@ -5,4 +5,4 @@ chemspipy==1.0.4
 pyparsing==2.0.5
 rdflib==4.2.1
 avogadro==1.92.0
--e git+git://github.com/alesgenova/cclib.git@fix-psi4-2#egg=cclib
+-e git+git://github.com/openchemistry/openchemistrypy.git@local-output#egg=openchemistry
```
Comment (cjh1) [inline requirements.txt]: We probably don't want to commit this change?

File: .circleci/config.yml (hunk 1 above)
Comment (alesgenova) [reply .circleci/config.yml]: We need it, or else this dependency from the openchemistry `setup.py` will not be satisfied
```python
    dependency_links=[
        'git+ssh://git@github.com/cclib/cclib.git#egg=cclib-1.6.1'
    ]
```

File: girder/molecules/server/calculation.py (hunk 2 above)
Comment (alesgenova) [reply girder/molecules/server/calculation.py]: Good point, I am now using the `SpooledTemporaryFile` class, that switches between `BytesIO` or `StringIO` to actual disk file when appropriate

Comment (cjh1) [reply girder/molecules/server/calculation.py]: Good plan

**Comment by cjh1** [comment]
@alesgenova Can you take a look at these failure?

**Pull request merged**